static/js/modules/maps-event.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var $ = require('jquery');
```

Comment:
Yay for pulling this out! Would it make more sense included in `maps.js` though? Fewer places to look in the future.